api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -597,6 +597,12 @@ public static String getOpenmrsProperty(String property) {
 	 * Specifies the uuid of the concept which represents drug non coded
 	 */
 	public static final String GP_DRUG_ORDER_DRUG_OTHER = "drugOrder.drugOther";
+	
+	/**
+	 * Name of property that stores the duration for which the password reset token is valid
+	 */
+	public static final String GP_PASSWORD_RESET_VALIDTIME = "validTime";
```

Comment:
There is a section for  other security related constants, please move this next to them and add a prefix similar to those other security constants